website/src/documentation/transforms/python/element-wise/tostring.md

```diff
@@ -19,9 +19,38 @@ limitations under the License.
 -->
 
 # ToString
-Transforms every element in an input collection a string.
 
-## Examples
-See [BEAM-7389](https://issues.apache.org/jira/browse/BEAM-7389) for updates. 
+<script type="text/javascript">
+localStorage.setItem('language', 'language-py')
+</script>
 
-## Related transforms 
\ No newline at end of file
+Transforms every element in an input collection to a string.
+
+## Example
+
+Any non-string element can be converted to a string using standard Python functions and methods.
+Many I/O transforms, such as `TextIO`, expect their input elements to be strings.
```

Comment:
Do you mean `textio.WriteToText` instead of `TextIO` ? Because these redoes not apply to read transforms.